Source/WebCore/css/parser/CSSParserFastPaths.cpp

```diff
@@ -649,7 +648,7 @@ static bool parseTransformTranslateArguments(CharType*& pos, CharType* end, unsi
         unsigned argumentLength = static_cast<unsigned>(delimiter);
         CSSUnitType unit = CSSUnitType::CSS_NUMBER;
         double number;
-        if (!parseSimpleLength(pos, argumentLength, unit, number))
+        if (!parseSimpleLength(std::span<const CharType> { pos, argumentLength }, unit, number))
```

Comment:
Is the template argument just so we can add const? I would have thought we could do this with just braces, or just `std::span` with no template argument.

Reply:
My first attempt was also just curly brackets. However, specifying the span template parameter was indeed required to add the `const`.